packages/web/src/containers/NavBarContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import NavBar from '../components/NavBar';
+
+const NavBarContainer = () => {
+  const username = 'Julio Cezar';
+  const [menuAnchorEl, setMenuAnchorEl] = React.useState<null | HTMLElement>(null);
+  const openMenu = Boolean(menuAnchorEl);
+
+  const handleMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setMenuAnchorEl(event.currentTarget);
+  };
+
+  const handleCloseMenu = () => {
+    setMenuAnchorEl(null);
+  }
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    const { id } = event.currentTarget;
+    if (id === "config") {
+      setMenuAnchorEl(null);
+      console.log('entrar em config')
+    }
+    if (id === "logout") {
+      setMenuAnchorEl(null);
+      console.log('sair');
+    }
+    setMenuAnchorEl(null);
```

Comment:
Precisa desse set aqui mesmo? Se o id for sempre config ou logout, creio que não precise. Se tiver outro valor, aí não precisa alterar

Reply:
e inclusive n tem return aqui entao ele vai rodar o setMenuAnchorEl 2x, dentro do if e fora

Reply:
Não entendi pq optou por esta função para fechar o menu, pq não fazer um handleClose igual o exemplo no próprio material UI https://material-ui.com/pt/components/menus/ ?

Reply:
Eu fiz um mix de todos os comentarios aqui, eu agrupei tudo numa mesma funcao chamada handleClose e coloquei o return null dentro dos ifs para que não chame o setMenuAnchorEl mais de uma vez. Eu fiz isso porque realmente nao precisaria de 2 funcoes uma pra click e uma pra close, uma vez, que eu posso tratar o close e o click dentro da mesma função, igual o Bolivar falou ali.